adapters/facebook.go

```diff
@@ -49,8 +51,21 @@ type facebookParams struct {
 	PlacementId string `json:"placementId"`
 }
 
+func coinFlip() bool {
+	random := rand.Intn(2)
+	if random == 0 {
+		return false
+	}
+	return true
+}
```

Comment:
Any reason we can't shorten this to `return rand.Intn(2) != 0`?

Reply:
nope, good suggestion.